ActionResource.ERROR_0001_FEATURE_NOT_IMPLEMENTED=Recurso ainda n\u00e3o implementado.

Base.CODE_COMPONENT_ID={0}:{1}
Base.CODE_LOG_ID={0}:COMPONENT:{1}:{2}
Base.DEBUG_VALIDATING_COMPONENT=Validando componente para a a\u00e7\u00e3o {0}
Base.DEBUG_VALIDATION_RESULT=executar a valida\u00e7\u00e3o=
Base.ERROR_0001_INVALID_SESSION=Sess\u00e3o inv\u00e1lida
Base.ERROR_0002_EXECUTION_FAILED=Componente de execu\u00e7\u00e3o falhou
Base.ERROR_0004_VALIDATION_FAILED=Componente de valida\u00e7\u00e3o falhou
Base.WARNING_LOGGING_LEVEL_UNKNOWN=N\u00edvel de logging desconhecido: depurar cria\u00e7\u00e3o

BaseRequestHandler.ERROR_0001_NO_SOLUTION_ENGINE=Solution Engine n\u00e3o pode ser criado

ComponentBase.ERROR_0003_INPUT_PARAM_MISSING=O par\u00e2metro de entrada necess\u00e1rio "{0}" n\u00e3o foi especificado no action-inputs
ComponentBase.ERROR_0004_OUTPUT_PARAM_MISSING=O par\u00e2metro de sa\u00edda necess\u00e1rio "{0}" n\u00e3o foi especificado no action-outputs
ComponentBase.ERROR_0005_RESOURCE_PARAM_MISSING=O recurso necess\u00e1rio "{0}" n\u00e3o foi especificado no action-resources
ComponentBase.ERROR_0006_COMPONENT_SETTING_PARAM_MISSING=A configura\u00e7\u00e3o necess\u00e1ria "{0}" n\u00e3o foi especificada no component-definition

DSAdmin.USER_CONNECTION_FAILED=Ocorreu um erro. A mensagem retornada foi {0}
DSAdmin.USER_CONNECTION_SUCCESS=Conex\u00e3o foi bem sucedida
DSAdmin.USER_DATASOURCE_NOT_FOUND=Erro: Datasource n\u00e3o foi encontrado
DSAdminHelper.USER_DATASOURCE_DELETED=O datasource solicitado {0} foi deletado
DSAdminHelper.USER_DATASOURCE_MISSING=O datasource solicitado {0} n\u00e3o foi encontrado
DSAdminHelper.USER_DATASOURCE_RENAMED=O datasource solicitado {0} foi renomeado
DSAdminHelper.USER_DATASOURCE_SAVED=O datasource solicitado {0} foi salvo
DSAdminHelper.USER_FILE_ERROR=Um arquivo n\u00e3o p\u00f4de ser encontrado ou modificado
DSAdminHelper.USER_OPERATION_FAILED=A opera\u00e7\u00e3o falhou
DSAdminHelper.USER_OPERATION_NOT_ALLOWED=N\u00e3o foi autorizada a conclus\u00e3o

JBossDSAdmin.ERROR_0001_DS_FILE_MISSING=Datasource {0} em jboss-web.xml n\u00e3o tem um datasource (-ds.xml) arquivo definido no diret\u00f3rio de deploy
JBossDSAdmin.ERROR_0002_COULD_NOT_DELETE=N\u00e3o foi poss\u00edvel excluir o arquivo do datasource: {0}
JBossDSAdmin.ERROR_0003_JBOSS_WEB_XML_NOT_FOUND=jboss-web.xml n\u00e3o foi encontrado em: {0}
JBossDSAdmin.ERROR_0004_JBOSS_WEB_XML_INVALID=jboss-web.xml n\u00e3o foi analisado: {0}
JBossDSAdmin.ERROR_0005_DS_FILE_INVALID=jboss -ds.xml n\u00e3o p\u00f4de ser analisado: {0}
JBossDSAdmin.ERROR_0006_ERRORSAVINGEDIT=Erro Salvando edi\u00e7\u00e3o para datasource JBoss
JBossDSAdmin.ERROR_0007_ERRORCREATINGDS=Erro criando datasource JBoss

Message.USER_DEBUG=Debug: {0} ({1})
Message.USER_ERROR=Error: {0} ({1})
Message.USER_ERROR_EX=Error: {0} - {2} ({1})
Message.USER_INFO=Info: {0} ({1})
Message.USER_WARNING=Aviso: {0} ({1})

ParameterManager.ERROR_0001_DISPOSE_ERROR=Descarte de par\u00e2metro de tempo de execu\u00e7\u00e3o {0} falhou.

RuntimeContext.CODE_XFORM_CONTROL_LABEL_END=</td></tr>
RuntimeContext.CODE_XFORM_CONTROL_LABEL_START=<tr><td class="portlet-font">{0}</td><td class="portlet-font">
RuntimeContext.DEBUG_AUDIT=audit: instanceId={0}, objectId={1}, messageType={2}
RuntimeContext.DEBUG_EXECUTING_ACTION=Execu\u00e7\u00e3o de defini\u00e7\u00e3o da a\u00e7\u00e3o: Intera\u00e7\u00e3o {0}
RuntimeContext.DEBUG_EXECUTING_ACTIONS=Execu\u00e7\u00e3o de seq\u00fc\u00eancia da a\u00e7\u00e3o
RuntimeContext.DEBUG_FINISHED_COMPONENT_EXECUTE=executeComponent finished audited execute
RuntimeContext.DEBUG_INITIALIZING_COMPONENT=Inicializando componentes
RuntimeContext.DEBUG_NO_ACTION=Nenhuma a\u00e7\u00e3o
RuntimeContext.DEBUG_PARAMETER_HTML=createParameterPage html={0}
RuntimeContext.DEBUG_POST-EXECUTE_AUDIT=execute post-audit
RuntimeContext.DEBUG_PRE-EXECUTE_AUDIT=execute pre-audit
RuntimeContext.DEBUG_SETTING_LOGGING=Definir o n\u00edvel de log de \u200b\u200bcomponentes para {0}
RuntimeContext.DEBUG_STARTING_COMPONENT_EXECUTE=executeComponent starting audited execute
RuntimeContext.DEBUG_VALIDATING_COMPONENT=validateComponent validando componente {0}
RuntimeContext.ERROR_0001_RUNTIME_RUNNING=Tempo de execu\u00e7\u00e3o j\u00e1 est\u00e1 em execu\u00e7\u00e3o.
RuntimeContext.ERROR_0002_ACTION_NOT_VALIDATED=A\u00e7\u00e3o do documento n\u00e3o v\u00e1lida: {0}
RuntimeContext.ERROR_0005_ACTION_RESOURCES_NOT_VALID=Recursos de a\u00e7\u00e3o n\u00e3o validam: {0}
RuntimeContext.ERROR_0006_ACTION_COMPONENT_NOT_VALID=Componentes de a\u00e7\u00e3o n\u00e3o validam: {0}
RuntimeContext.ERROR_0008_ACTION_INITIALIZATION_FAILED=Falha na a\u00e7\u00e3o de inicializa\u00e7\u00e3o: {0}
RuntimeContext.ERROR_0009_COULD_NOT_CREATE_COMPONENT=N\u00e3o foi poss\u00edvel criar componente {0}
RuntimeContext.ERROR_0010_RUNTIME_DID_NOT_VALIDATE=Tempo de execu\u00e7\u00e3o n\u00e3o foi validado
RuntimeContext.ERROR_0011_NO_VALID_ACTIONS=ActionDefinitions n\u00e3o v\u00e1lidos dispon\u00edveis
RuntimeContext.ERROR_0012_EXECUTION_FAILED=ActionDefinition para {0} n\u00e3o foi executado com sucesso
RuntimeContext.ERROR_0013_BAD_PARAMETERS=Par\u00e2metros n\u00e3o p\u00f4de ser resolvido
RuntimeContext.ERROR_0014_NO_OUTPUT_HANDLER=Manipulador de sa\u00edda n\u00e3o p\u00f4de ser resolvido
RuntimeContext.ERROR_0016_COMPONENT_INITIALIZE_FAILED=inicializa\u00e7\u00e3o do componente falhou
RuntimeContext.ERROR_0017_COMPONENT_EXECUTE_FAILED=Falha na a\u00e7\u00e3o de execu\u00e7\u00e3o
RuntimeContext.ERROR_0018_PARAMETER_NOT_FULFILLED=O par\u00e2metro solicitado "{0}" tem um valor nulo
RuntimeContext.ERROR_0019_INVALID_INPUT_REQUEST=Par\u00e2metro de entrada inv\u00e1lido "{0}" solicitado durante a execu\u00e7\u00e3o de {1}
RuntimeContext.ERROR_0020_INVALID_LOOP_PARAMETER=Par\u00e2metro de loop inv\u00e1lido "{0}" solicitado durante a execu\u00e7\u00e3o de {1}
RuntimeContext.ERROR_0021_INVALID_OUTPUT_REQUEST=Par\u00e2metro de sa\u00edda inv\u00e1lido "{0}" solicitado durante a execu\u00e7\u00e3o de {1}
RuntimeContext.ERROR_0022_INVALID_RESOURCE_REQUEST=Recurso inv\u00e1lido "{0}" solicitado durante a execu\u00e7\u00e3o de {1}
RuntimeContext.ERROR_0023_INVALID_OUTPUT_STREAM=Fluxo de sa\u00edda solicitado n\u00e3o pode ser fornecido para "{0}" porque n\u00e3o \u00e9 do tipo 'content'
RuntimeContext.ERROR_0024_NO_CONTENT_REPOSITORY=N\u00e3o foi poss\u00edvel obter reposit\u00f3rio de conte\u00fado
RuntimeContext.ERROR_0027_COULD_NOT_CREATE_CHILD=N\u00e3o foi poss\u00edvel criar inst\u00e2ncia child
RuntimeContext.ERROR_0029_SAVE_PARAM_NOT_FOUND=Erro par\u00e2metro de sa\u00edda "{0}" n\u00e3o foi encontrado salvo.
RuntimeContext.ERROR_0030_SEND_FEEDBACKFORM=N\u00e3o foi poss\u00edvel criar formul\u00e1rio de feedback.
RuntimeContext.ERROR_0031_INPUT_NOT_FOUND=O par\u00e2metro necess\u00e1rio"{0}" n\u00e3o foi especificado no action-inputs
RuntimeContext.ERROR_0032_CONDITIONAL_EXECUTION_FAILED=A execu\u00e7\u00e3o condicional n\u00e3o foi conclu\u00edda com \u00eaxito.
RuntimeContext.ERROR_0033_NOT_PEEKABLE=O conjunto de resultados especificado para looping n\u00e3o \u00e9 peekable mas, "peek-only' foi criado
RuntimeContext.ERROR_0034_IO_ERROR=Ocorreu um erro de sa\u00edda para o fluxo.
RuntimeContext.ERROR_0035_ACTION_VALIDATION_FAILED=A\u00e7\u00e3o de valida\u00e7\u00e3o falhou.
RuntimeContext.INFO_ACTION_NOT_EXECUTED=A\u00e7\u00f5es n\u00e3o executadas devido \u00e0 condi\u00e7\u00e3o.
RuntimeContext.TRACE_INFO_USING_CURRENT_PARAMETER_VALUE=usando o valor atual de "{0}"
RuntimeContext.TRACE_USING_DEFAULT_PARAMETER_VALUE=utilizando o valor padr\u00e3o de "{0}"
RuntimeContext.USER_BAD_PARAMETER_PAGE=N\u00e3o foi poss\u00edvel gerar a p\u00e1gina de par\u00e2metro
RuntimeContext.USER_PARAMETER_FORM_SUBMIT=Submeter
RuntimeContext.WARN_UNRECOGNIZED_SCOPE=Escopo {0} n\u00e3o \u00e9 reconhecido para o par\u00e2metro {1}.
RuntimeContext.WARN_DEPRECATED_COMPONENT_CLASS=Tentativa de uso, em vez de {0}, por favor use {1}
RuntimeContext.WARN_REQUESTED_PARAMETER_SOURCE_NOT_AVAILABLE=O provedor de par\u00e2metro solicitado {0} por par\u00e2metro {1} n\u00e3o est\u00e1 dispon\u00edvel
RuntimeContext.WARN_VARIABLE_IN_LOOP_IS_NULL=Vari\u00e1vel em loop \u00e9 nulo: "{0}"
RuntimeContext.WARN_VARIABLE_IN_LOOP_NOT_RECOGNIZED=Vari\u00e1vel em loop n\u00e3o \u00e9 reconhecida: {0} do tipo {1}
RuntimeContext.INFO_NO_OUTPUT_HANDLER=Sa\u00edda de "resposta" foi presente, mas manipulador de sa\u00edda era nulo.
RuntimeContext.ERROR_UNABLE_TO_GET_RESOURCE_AS_DOCUMENT=N\u00e3o \u00e9 poss\u00edvel obter um recurso como documento
RuntimeContext.WARN_NO_PLUGIN_PROPERTIES_BUNDLE=N\u00e3o foi poss\u00edvel ler plugin.properties do pacote de tempo de execu\u00e7\u00e3o.
RuntimeContext.WARN_NO_PLUGIN_PROPERTIES=No override plugin properties found in the system solution
RuntimeContext.WARN_BAD_PLUGIN_PROPERTIES=Exception reading override properties

SoapHelper.ERROR_0001_UNKNOWN_ERROR=Erro desconhecido

SequenceDefinition.DEBUG_ADDING_SOURCE_FOR_PARAMETER=Acrescentar uma fonte vari\u00e1vel {0} para par\u00e2metro {1}
SequenceDefinition.DEBUG_USING_DEFAULT_VALUE=Adicionando valor padr\u00e3o de "{0}" para par\u00e2metro {1}
SequenceDefinition.ERROR_0002_NO_ACTION_SEQUENCE_NODE=ActionSequenceFactory XML inv\u00e1lido: 'action-sequence' n\u00f3 n\u00e3o encontrado em {0}/{1}/{2}
SequenceDefinition.ERROR_0004_VARIABLE_SOURCE_NOT_VALID=Defini\u00e7\u00e3o de vari\u00e1vel {0} para par\u00e2metro {1} n\u00e3o est\u00e1 v\u00e1lido
SequenceDefinition.ERROR_0005_PARSING_PARAMETERS=Analisar par\u00e2metros de seq\u00fc\u00eancia de a\u00e7\u00e3o
SequenceDefinition.ERROR_0006_PARSING_RESOURCE=Analisando recurso na sequ\u00eancia de a\u00e7\u00e3o
SequenceDefinition.ERROR_0007_RESOURCE_NO_MIME_TYPE=Recurso chamado {0} foi ignorado porque n\u00e3o h\u00e1 nenhum tipo mimo
SequenceDefinition.ERROR_0008_RESOURCE_NO_LOCATION=Recurso chamado {0} foi ignorado porque n\u00e3o h\u00e1 local especificado
SequenceDefinition.INFO_OUTPUT_PARAMETERS_NOT_DEFINED=N\u00e3o h\u00e1 par\u00e2metros de sa\u00edda definidos no Documento de Action Sequence.
SequenceDefinition.INFO_RESOURCES_PARAMETERS_NOT_DEFINED=N\u00e3o h\u00e1 par\u00e2metros de recursos definidos no Documento de Action Sequence.

ServerDSAdmin.ERROR_0001_USER_WEB_XML_NOT_FOUND=web.xml n\u00e3o encontrada em: {0}
ServerDSAdmin.ERROR_0002_WEB_XML_INVALID=web.xml n\u00e3o foi poss\u00edvel ser analisado: {0}
ServerDSAdmin.ERROR_0007_DATASOURCE_IS_NULL=N\u00e3o \u00e9 poss\u00edvel salvar um datasource nulo

SolutionEngine.DEBUG_GETTING_RUNTIME_CONTEXT=Obtendo contexto de execu\u00e7\u00e3o e dos dados
SolutionEngine.DEBUG_LOADING_ACTION_DEFINITION=Carregando a\u00e7\u00e3o de arquivo de defini\u00e7\u00e3o de seq\u00fc\u00eancia
SolutionEngine.DEBUG_STARTING_EXECUTION=Iniciando execu\u00e7\u00e3o de {0}
SolutionEngine.ERROR_0001_PROCESS_NOT_SPECIFIED=Nenhum processo especificado
SolutionEngine.ERROR_0002_SOLUTION_NOT_SPECIFIED=Nenhuma solu\u00e7\u00e3o especificada
SolutionEngine.ERROR_0003_PATH_NOT_SPECIFIED=Nenhuma solu\u00e7\u00e3o especificada para o caminho
SolutionEngine.ERROR_0004_ACTION_NOT_SPECIFIED=Nenhuma seq\u00fc\u00eancia de a\u00e7\u00e3o especificada
SolutionEngine.ERROR_0006_ACTION_SEQUENCE_INVALID=Valida\u00e7\u00e3o da a\u00e7\u00e3o de sequencia falhou
SolutionEngine.ERROR_0007_ACTION_EXECUTION_FAILED=Execu\u00e7\u00e3o da a\u00e7\u00e3o de seq\u00fc\u00eancia falhou
SolutionEngine.ERROR_0008_INVALID_INSTANCE=Id inv\u00e1lido na inst\u00e2ncia solicitada "{0}"


SolutionPublisher.ERROR_0001_PUBLISH_FAILED=N\u00e3o foi poss\u00edvel publicar no Solution Repository
SolutionPublisher.USER_ERROR_PUBLISH_FAILED=O editor encontrou erros no Solutions Repository:
SolutionPublisher.USER_SOLUTION_REPOSITORY_UPDATED=O Solution Repository foi atualizado

SolutionRepository.USER_PUBLISH_DESCRIPTION=Leia todos os arquivos de solu\u00e7\u00e3o e volte a gerar o \u00edndice de reposit\u00f3rio
SolutionRepository.USER_PUBLISH_TITLE=Atualizar Solution Repository

Template.ERROR_0005_COULD_NOT_DETERMINE_COLUMN=Coluna no conjunto de resultados n\u00e3o p\u00f4de ser determinado. Para formatar usando um conjunto de resultados, verifique se o nome da coluna e nome de entrada s\u00e3o os mesmos, ou certifique-se o conjunto de resultados tem apenas uma coluna.


MessageFormatter.ERROR_0001_REQUEST_FAILED=A solicita\u00e7\u00e3o falhou
MessageFormatter.ERROR_0002_COULD_NOT_PROCESS=O motor de solu\u00e7\u00e3o n\u00e3o p\u00f4de processar o evento
MessageFormatter.ERROR_0003_NO_TEMPLATE_FOUND=Falha ao obter um modelo HTML para esta resposta de erro em {0}
MessageFormatter.USER_ACTION_SUCCESSFUL=A\u00e7\u00e3o de sucesso
MessageFormatter.USER_SERVER_VERSION=Server Version: {0}
MessageFormatter.USER_START_ACTION=Plataforma Pentaho BI - Comece a a\u00e7\u00e3o
MessageFormatter.ACTION_SEQUENCE_EXECUTION_FAILED=Action Sequence falha na execu\u00e7\u00e3o, veja os detalhes abaixo
MessageFormatter.EXCEPTION_STACK_TRACE=Rastreamento de pilha:
MessageFormatter.EXCEPTION_FIELD_NOT_APPLICABLE=N/A
MessageFormatter.LOG_EXCEPTION_TIME=| Error Time: {0}
MessageFormatter.LOG_EXCEPTION_SESSION_ID=| Sess\u00e3o ID: {0}
MessageFormatter.LOG_EXCEPTION_INSTANCE_ID=| Inst\u00e2ncia Id: {0}
MessageFormatter.LOG_EXCEPTION_ACTION_SEQUENCE=| Action Sequence: {0}
MessageFormatter.LOG_EXCEPTION_ACTION_CLASS=| Action Class: {0}
MessageFormatter.LOG_EXCEPTION_ACTION_DESC=| Action Desc: {0}
MessageFormatter.LOG_EXCEPTION_STEP_NUM=| Step #: {0}
MessageFormatter.LOG_EXCEPTION_LOOP_INDEX=| Loop Index: {0}
MessageFormatter.LOG_EXCEPTION_ACTION_SEQUENCE_EXECUTION_STACK=| Execu\u00e7\u00e3o da pilha:
MessageFormatter.RESPONSE_ERROR_HEADING=A plataforma de BI Pentaho relatou um erro durante a execu\u00e7\u00e3o de uma seq\u00fc\u00eancia de a\u00e7\u00e3o
MessageFormatter.RESPONSE_EXCEPTION_TIME_LABEL=Error Time:
MessageFormatter.RESPONSE_EXCEPTION_MESSAGES_LABEL=Causas poss\u00edveis:
MessageFormatter.RESPONSE_EXCEPTION_MSG_LABEL=Mesagem de erro:
MessageFormatter.RESPONSE_EXCEPTION_TYPE_LABEL=Tipo de erro:
MessageFormatter.RESPONSE_EXCEPTION_SESSION_ID_LABEL=Sess\u00e3o ID:
MessageFormatter.RESPONSE_EXCEPTION_INSTANCE_ID_LABEL=Inst\u00e2ncia Id:
MessageFormatter.RESPONSE_EXCEPTION_ACTION_SEQUENCE_LABEL=Action Sequence:
MessageFormatter.RESPONSE_EXCEPTION_ACTION_SEQUENCE_EXECUTION_STACK_LABEL=Execu\u00e7\u00e3o da pilha:
MessageFormatter.RESPONSE_EXCEPTION_ACTION_CLASS_LABEL=Action Class:
MessageFormatter.RESPONSE_EXCEPTION_ACTION_DESC_LABEL=Descri\u00e7\u00e3o da a\u00e7\u00e3o:
MessageFormatter.RESPONSE_EXCEPTION_STEP_NUM_LABEL=Passo #:
MessageFormatter.RESPONSE_EXCEPTION_LOOP_INDEX_LABEL=Loop Index:
MessageFormatter.RESPONSE_EXCEPTION_STACK_TRACE_LABEL=Rastreamento de pilha:

PRO_SUBSCRIPTREP.ERROR_0005_GENERAL_ERROR=Erro na Assinatura do Reposit\u00f3rio

CacheManager.ERROR_0001_NOSESSION=N\u00e3o pode colocar em cache sem um ID de sess\u00e3o v\u00e1lido
CacheManager.ERROR_0002_NOT_INSTANCE_OF_CACHE_PROVIDER=Objeto n\u00e3o inst\u00e2ncia de provedor de cache
CacheManager.ERROR_0003_CACHE_NOT_ENABLED=Cache n\u00e3o est\u00e1 habilitado
CacheManager.ERROR_0005_UNABLE_TO_BUILD_CACHE=Incapaz de construir o cache
CacheManager.WARN_0001_CACHE_NOT_ENABLED=Cache n\u00e3o est\u00e1 habilitado
CacheManager.WARN_0002_REGION_ALREADY_EXIST=Regi\u00e3o j\u00e1 existe {0}
CacheManager.WARN_0003_REGION_DOES_NOT_EXIST=Regi\u00e3o n\u00e3o existe {0}
CacheManager.INFO_0001_CACHE_DOES_NOT_EXIST=Cache n\u00e3o existe

ApacheVFSOutputHandler.ERROR_0001_CANNOT_GET_VFSMGR=N\u00e3o \u00e9 poss\u00edvel obter VFS FileSystemManager
ApacheVFSOutputHandler.ERROR_0002_CANNOT_GET_VF=N\u00e3o \u00e9 poss\u00edvel obter arquivos virtuais: {0}
ApacheVFSOutputHandler.ERROR_0003_CANNOT_WRITE=N\u00e3o \u00e9 poss\u00edvel gravar arquivo Virtual: {0}
ApacheVFSOutputHandler.ERROR_0004_CANNOT_GET_CTX=N\u00e3o \u00e9 poss\u00edvel obter o conte\u00fado do arquivo Virtual: {0}
ApacheVFSOutputHandler.ERROR_0005_CANNOT_GET_HANDLER=Erro ao obter manipulador de sa\u00edda para {0}

MetadataPublisher.USER_PUBLISHER_NAME=Relat\u00f3rios de metadados
MetadataPublisher.USER_PUBLISHER_DESCRIPTION=Metadados usados para relat\u00f3rios adhoc
MetadataPublisher.USER_METADATA_RELOADED=Relat\u00f3rios de metadados para {0} solu\u00e7\u00e3o(s) foram recarregados.
MetadataPublisher.USER_DELETE_META_FAILED=Problemas ocorreram ao apagar arquivos antigos metadata.xmi.
MetadataPublisher.ERROR_0001_USER_IMPORT_META_FAILED=Problemas ocorreram ao carregar alguns arquivos metadata.xmi
MetadataPublisher.INFO_DELETING_METADATA=Excluindo os metadados existentes para {0}
MetadataPublisher.INFO_0001_CHECK_LOG=Verifique o log do servidor
MetadataPublisher.DEBUG_ALREADY_LOADED=Metadados j\u00e1 foram carregados para {0}
MetadataPublisher.INFO_IMPORTING_METADATA=Importando metadados para {0}

PMDSystemListener.ERROR_0001_PROPERTIES_NOT_FOUND=N\u00e3o foi poss\u00edvel encontrar properties para inicializar o reposit\u00f3rio de metadados - as consultas de metadados n\u00e3o funcionar\u00e3o
PMDSystemListener.ERROR_0002_COULD_NOT_INITIALIZE=N\u00e3o foi poss\u00edvel inicializar o reposit\u00f3rio de metadados - as consultas de metadados n\u00e3o funcionar\u00e3o

TemplateUtil.NOT_FOUND=N\u00e3o foi encontrado: {0}
TemplateUtil.INVALID_COLUMN=Coluna inv\u00e1lida: {0}
TemplateUtil.NO_TOKEN=no tokens detected

StandaloneSimpleJNDIDatasourceAdmin.ERROR_0001_SOLUTION_REPO_NOT_FOUND=Solution repository n\u00e3o foi encontrado em {0}
StandaloneSimpleJNDIDatasourceAdmin.ERROR_0002_INPUT_STREAM_NOT_FOUND=N\u00e3o foi poss\u00edvel obter fluxo de entrada {0}
StandaloneSimpleJNDIDatasourceAdmin.ERROR_0003_INPUT_STREAM_COULD_NOT_BE_CLOSED=N\u00e3o foi poss\u00edvel fechar fluxo de entrada
StandaloneSimpleJNDIDatasourceAdmin.ERROR_0004_OUTPUT_STREAM_COULD_NOT_BE_CLOSED=N\u00e3o foi poss\u00edvel fechar fluxo de sa\u00edda

MondrianCatalogPublisher.ERROR_0005_PUBLISH_EXCEPTION=Exce\u00e7\u00e3o ocorreu durante a publica\u00e7\u00e3o

PooledDatasourceSystemListener.ERROR_0001_UNABLE_TO_INSTANTIATE_OBJECT=Imposs\u00edvel instanciar objeto
PooledDatasourceSystemListener.ERROR_0002_UNABLE_TO_GET_DATASOURCE=N\u00e3o foi poss\u00edvel obter objeto do data source
PooledDatasourceSystemListener.ERROR_0003_UNABLE_TO_POOL_DATASOURCE=Incapaz de pegar todos os objetos do datasource: {0} causado pelo {1}
NonPooledDatasourceSystemListener.ERROR_0001_UNABLE_TO_INSTANTIATE_OBJECT=Imposs\u00edvel instanciar objeto
NonPooledDatasourceSystemListener.ERROR_0002_UNABLE_TO_GET_DATASOURCE=N\u00e3o foi poss\u00edvel obter objeto do data source
NonPooledDatasourceSystemListener.ERROR_0003_UNABLE_TO_POOL_DATASOURCE=Incapaz de pegar todos os objetos do datasource: {0} causado pelo {1}

JndiDatasourceService.ERROR_0003_UNABLE_TO_GET_JNDI_DATASOURCE=Imposs\u00edvel de pegar jndi do datasource
NonPooledDatasourceService.ERROR_0001_UNABLE_TO_INSTANTIATE_OBJECT=Imposs\u00edvel instanciar objeto
NonPooledDatasourceService.ERROR_0002_UNABLE_TO_GET_DATASOURCE=N\u00e3o foi poss\u00edvel obter objeto do data source
NonPooledDatasourceService.ERROR_0003_UNABLE_TO_GET_JNDI_DATASOURCE=Imposs\u00edvel de pegar jndi do datasource
PooledDatasourceService.ERROR_0001_UNABLE_TO_INSTANTIATE_OBJECT=Imposs\u00edvel instanciar objeto
PooledDatasourceService.ERROR_0002_UNABLE_TO_GET_DATASOURCE=N\u00e3o foi poss\u00edvel obter objeto do data source
PooledDatasourceService.ERROR_0003_UNABLE_TO_GET_JNDI_DATASOURCE=Imposs\u00edvel de pegar jndi do datasource
NonPooledOrJndiDatasourceService.ERROR_0001_UNABLE_TO_INSTANTIATE_OBJECT=Imposs\u00edvel instanciar objeto
NonPooledOrJndiDatasourceService.ERROR_0002_UNABLE_TO_GET_DATASOURCE=N\u00e3o foi poss\u00edvel obter objeto do data source
NonPooledOrJndiDatasourceService.ERROR_0003_UNABLE_TO_GET_JNDI_DATASOURCE=Imposs\u00edvel de pegar jndi do datasource
PooledOrJndiDatasourceService.ERROR_0001_UNABLE_TO_INSTANTIATE_OBJECT=Imposs\u00edvel instanciar objeto
PooledOrJndiDatasourceService.ERROR_0002_UNABLE_TO_GET_DATASOURCE=N\u00e3o foi poss\u00edvel obter objeto do data source
PooledOrJndiDatasourceService.ERROR_0003_UNABLE_TO_GET_JNDI_DATASOURCE=Imposs\u00edvel de pegar jndi do datasource

PentahoConnectionFactory.ERROR_0001_COULD_NOT_CREATE_CONNECTION=N\u00e3o foi poss\u00edvel criar IPentahoConnection [{0}]

SimpleContentGenerator.ERROR_0001_NO_OUTPUT_HANDLER=N\u00e3o foi poss\u00edvel obter manipulador de sa\u00edda
SimpleContentGenerator.ERROR_0002_NO_CONTENT_ITEM=N\u00e3o foi poss\u00edvel obter uma sa\u00edda do item de conte\u00fado
SimpleContentGenerator.ERROR_0003_NO_OUTPUT_STREAM=N\u00e3o foi poss\u00edvel obter fluxo de sa\u00edda

PojoComponent.CANNOT_USE_SETCLASS=Setter inv\u00e1lido - n\u00e3o pode chamar um m\u00e9todo chamado setClass
PojoComponent.UNUSED_INPUT=Par\u00e2metro de entrada n\u00e3o utilizado: {0}

ActionDelegate.ERROR_0001_MIMETYPE_NOT_DECLARED=MIME TYPE n\u00e3o pode ser nulo para uma a\u00e7\u00e3o de streaming
ActionDelegate.ERROR_0002_ACTION_CANNOT_ACCEPT_STREAM=Fluxo de sa\u00edda "{0}" encontrado, mas Action "{1}" n\u00e3o pode aceit\u00e1-lo.  Action deve implementar IStreamingAction.
ActionDelegate.ERROR_0003_OUTPUT_STREAM_NOT_AVAILABLE_1=N\u00e3o foi poss\u00edvel obter um fluxo de sa\u00edda para a sa\u00edda de tipo de conte\u00fado "{0}".
ActionDelegate.ERROR_0004_OUTPUT_STREAM_NOT_AVAILABLE_2=N\u00e3o foi poss\u00edvel obter um fluxo de sa\u00edda para a Action de streaming "{0}".  Causa: fluxo de sa\u00edda nulo do item de conte\u00fado
ActionDelegate.ERROR_0005_FAILED_TO_SET_INPUT=Erro de configura\u00e7\u00e3o de entrada"{0}".  Tentativa de definir a entrada de defini\u00e7\u00e3o da a\u00e7\u00e3o do tipo"{1}" em uma a\u00e7\u00e3o ("{2}") propriedade do tipo "{3}"
ActionDelegate.ERROR_0006_FAILED_TO_SET_RESOURCE=Recurso de configura\u00e7\u00e3o de erro nomeado "{0}".  Tentativa de definir recursos defini\u00e7\u00e3o da a\u00e7\u00e3o do tipo "{1}" em uma a\u00e7\u00e3o ({2}) propriedade do tipo "{3}"
ActionDelegate.ERROR_0007_NO_ACTION_BEAN_SPECIFIED=Nenhuma a\u00e7\u00e3o foi especificada para este bean ActionDelegate
ActionDelegate.ERROR_0008_FAILED_TO_SET_STREAM=Fluxo de configura\u00e7\u00e3o de erro para a sa\u00edda "{0}".  Tentativa de definir a sa\u00edda defini\u00e7\u00e3o da a\u00e7\u00e3o do tipo"{1}" em uma a\u00e7\u00e3o ("{2}") propriedade do tipo "{3}"
ActionDelegate.WARN_INPUT_NOT_WRITABLE=Action "{0}" n\u00e3o tem nenhum m\u00e9todo para a cria\u00e7\u00e3o de entrada nomeado"{1}" do tipo "{2}"
ActionDelegate.WARN_OUTPUT_NOT_READABLE=defini\u00e7\u00e3o de a\u00e7\u00f5es espera uma sa\u00edda nomeada "{0}" (do tipo "{1}"), mas a a\u00e7\u00e3o "{2}" n\u00e3o tem como acessar uma propriedade de bean com esse nome
ActionDelegate.WARN_RESOURCE_NOT_WRITABLE=Action "{0}" n\u00e3o tem nenhum m\u00e9todo para definir recursos"{1}" do tipo "{2}"
ActionDelegate.WARN_USING_IO_COMPATIBILITY_MODE=Voc\u00ea est\u00e1 usando de modo compatibilidade dos nomes dos argumentos de a\u00e7\u00e3o. O uso de tra\u00e7os de entradas, sa\u00eddas e recursos \u00e9 preterido em favor de camel-case.  Por favor use "{0}" em vez de "{1}".

SolutionRepoSaveContentGenerator.ERROR_0001_NO_FILEPATH=Nenhum par\u00e2metro 'filepath' fornecido
SolutionRepoSaveContentGenerator.ERROR_0002_NO_STATE=Nenhum par\u00e2metro "estado" fornecido
SolutionRepoSaveContentGenerator.ERROR_0003_BAD_PATH=Filepath fornecido \u00e9 inv\u00e1lido: {0}
SolutionRepoSaveContentGenerator.ERROR_0004_CANNOT_REPLACE=O arquivo j\u00e1 existe, mas par\u00e2metro "replace" n\u00e3o foi definido
SolutionRepoSaveContentGenerator.ERROR_0005_CREDENTIALS=Credenciais inv\u00e1lidas
SolutionRepoSaveContentGenerator.ERROR_0006_SAVE_FAILED=N\u00e3o foi poss\u00edvel salvar o arquivo de reposit\u00f3rio
SolutionRepoSaveContentGenerator.USER_FILE_SAVE=Arquivo salvo com sucesso

SolutionRepoLoadContentGenerator.ERROR_0001_LOAD_FAILED=Carga de arquivo falhou (arquivo ausente ou inv\u00e1lido): {0}

SolutionResourceContentGenerator.ERROR_0001_INVALID_PATH=Caminho inv\u00e1lido solicitado, o conte\u00fado deve estar em 'solution'/web: {0}
SolutionResourceContentGenerator.ERROR_0002_INVALID_MIMETYPE=MIME type n\u00e3o permitidos ou n\u00e3o conhecido por: {0}
SolutionResourceContentGenerator.ERROR_0003_RESOURCE_NOT_FOUND=Recurso n\u00e3o encontrado: {0}

SolutionURLContentGenerator.ERROR_0001_CANNOT_HANDLE_TYPE=N\u00e3o consegue lidar com este tipo de conte\u00fado: {0}
SolutionURLContentGenerator.ERROR_0002_RESOURCE_NOT_FOUND=Recurso n\u00e3o encontrado: {0}

SolutionURLContentGenerator.ERROR_0001_NO_FILEPATH=Nenhum par\u00e2metro 'path' fornecido
SolutionURLContentGenerator.ERROR_0002_CANNOT_HANDLE_TYPE=Tipo de conte\u00fado inv\u00e1lido: {0}
SolutionURLContentGenerator.ERROR_0003_RESOURCE_NOT_FOUND=Recurso n\u00e3o encontrado: {0}

TenantAwareDatasourceService.ERROR_0001_NAME_FORMAT_ILLEGAL=datasourceNameFormat ilegal - deve conter {0} e um {1} elemento.
TenantAwareDatasourceService.ERROR_0002_TENANT_ID_REQUIRED=Tenant id is required.
TenantAwareDatasourceService.ERROR_0003_SESSION_VARIABLE_NAME_REQUIRED=O tenantSessionVariableName deve ser configurado.

ActionSequenceJCRHelper.ERROR_0017_INVALID_XML_DOCUMENT=O documento xml \u00e9 inv\u00e1lido {0}
ActionSequenceJCRHelper.ERROR_0019_NO_DATA_IN_FILE=O arquivo n\u00e3o possui dados {0}
ActionSequenceJCRHelper.ERROR_0009_INVALID_DOCUMENT=O documento solution {0} n\u00e3o \u00e9 um documento XML v\u00e1lido
ActionSequenceJCRHelper.WARN_MISSING_RESOURCE_PROPERTY=Faltando propriedade "{0}" em arquivo de recurso "{1}" para "{2}"
ActionSequenceJCRHelper.ERROR_0007_COULD_NOT_READ_PROPERTIES=N\u00e3o foi poss\u00edvel ler as propriedades do arquivo {0}
ActionSequenceJCRHelper.ERROR_0001_INVALID_REPOSITORY=N\u00e3o foi poss\u00edvel recuperar um reposit\u00f3rio da sess\u00e3o
DatasourceService.DEBUG_0001_UNABLE_TO_FIND_DATASOURCE_IN_REPOSITORY=Imposs\u00edvel encontrar fonte de dados no reposit\u00f3rio gerenciado Pentaho: porque {0}
DatasourceSystemListener.WARN_0001_UNABLE_TO_GET_CONNECTION_ADDRESS=Imposs\u00edvel obter endere\u00e7o da conex\u00e3o. Suponho que hosts e portas sejam diferentes
DatasourceSystemListener.WARN_0002_UNABLE_TO_PARSE_SERVER_URL=N\u00e3o \u00e9 poss\u00edvel analisar URL do servidor (totalmente qualificado). Suponho que hosts e portas sejam diferentes
DatasourceSystemListener.WARN_0003_UNABLE_TO_GET_SERVER_ADDRESS=N\u00e3o \u00e9 poss\u00edvel obter URL do servidor. Suponho que hosts e portas sejam diferentes
DatasourceSystemListener.WARN_0004_UNABLE_TO_GET_PORT_NUMBER=Imposs\u00edvel obter n\u00famero da porta de conex\u00e3o. Suponho que hosts e portas sejam diferentes
PooledDatasourceHelper.ERROR_0001_DATASOURCE_CREATE_ERROR_NO_DIALECT=Imposs\u00edvel criar conex\u00e3o com a fonte de dados [ {0} ]. Nenhuma informa\u00e7\u00e3o de dialeto dispon\u00edvel.
PooledDatasourceHelper.ERROR_0002_DATASOURCE_CREATE_ERROR_NO_CLASSNAME=Imposs\u00edvel criar conex\u00e3o com a fonte de dados [ {0} ]. Nenhum driver com o nome da classe dispon\u00edvel.
PooledDatasourceHelper.ERROR_0003_DATASOURCE_CREATE_ERROR_NO_DRIVER=Imposs\u00edvel criar conex\u00e3o com a fonte de dados [ {0} ]. Imposs\u00edvel obter informa\u00e7\u00f5es do driver no dialeto.
PooledDatasourceHelper.ERROR_0004_UNABLE_TO_POOL_DATASOURCE_NO_DIALECT=Incapaz de reunir no pool de conex\u00f5es do banco de dados [ {0} ]. Nenhuma informa\u00e7\u00e3o no dialeto dispon\u00edvel.
PooledDatasourceHelper.ERROR_0005_UNABLE_TO_POOL_DATASOURCE_NO_DIALECT_SERVICE=Incapaz de reunir no pool de conex\u00f5es do banco de dados [ {0} ]. Nenhum dialeto dispon\u00edvel.
PooledDatasourceHelper.ERROR_0006_UNABLE_TO_POOL_DATASOURCE_NO_CLASSNAME=Incapaz de reunir no pool de conex\u00f5es do banco de dados  [ {0} ]. Nenhum driver com o nome da classe dispon\u00edvel.
PooledDatasourceHelper.ERROR_0007_UNABLE_TO_POOL_DATASOURCE_NO_DRIVER=Incapaz de reunir no pool de conex\u00f5es do banco de dados [ {0} ]. Imposs\u00edvel obter informa\u00e7\u00f5es do driver no dialeto.
PooledDatasourceHelper.ERROR_0008_UNABLE_TO_POOL_DATASOURCE_IT_IS_JNDI=Incapaz de reunir Fonte de Dados [ {0} ]. \u00c9 a conex\u00e3o JNDI.
PooledDatasourceHelper.ERROR_0009_UNABLE_TO_POOL_DATASOURCE_CANT_INITIALIZE=Unable to pool Data Source [ {0} ]. Cannot initialize {1}.
PooledDatasourceHelper.ERROR_0001_DATASOURCE_CANNOT_LOAD_DIALECT_SVC=Unable to load IDatabaseDialectService from the Pentaho session.
